static/app/views/replays/list/replayOnboardingPanel.tsx

```diff
@@ -141,6 +144,81 @@ export function SetupReplaysCTA({
   orgSlug,
 }: SetupReplaysCTAProps) {
   const {activateSidebar} = useReplayOnboardingSidebarPanel();
+  const [expanded, setExpanded] = useState(0);
+  const FAQ = [
+    {
+      header: () => (
+        <QuestionContent>{t('Can I use Session Replay with my app?')}</QuestionContent>
+      ),
+      content: () => (
+        <AnswerContent>
+          <div>
+            {t(
+              'Session Replay supports all browser-based applications. This includes static websites, single-page-aplications, and also server-side-rendered. The only prerequisite is that your application uses our Sentry JavaScript SDK (7.2.0 or greater) either with NPM/Yarn or our JS Loader script.'
+            )}
+          </div>
+          <div>
+            {tct(
+              'To learn about which SDKs we support, please visit [link:this page] from our docs.',
```

Comment:
For these links, should we shorten it to something like:

`..., please visit [our docs]`